File: CS/PopupSizeExample.Module/DatabaseUpdate/Updater.cs
using System;

using DevExpress.ExpressApp;
using DevExpress.ExpressApp.Updating;
using DevExpress.Xpo;
using DevExpress.Data.Filtering;
using DevExpress.Persistent.BaseImpl;
using DevExpress.ExpressApp.Security;
using PopupSizeExample.Module.BusinessObjects;

namespace PopupSizeExample.Module.DatabaseUpdate {
    public class Updater : ModuleUpdater {
        public Updater(IObjectSpace objectSpace, Version currentDBVersion) : base(objectSpace, currentDBVersion) { }
        public override void UpdateDatabaseAfterUpdateSchema() {
            base.UpdateDatabaseAfterUpdateSchema();
            for (byte i = 0; i <= 10; i++) {
                var width = i * 40 + 600;
                var height = i * 20 + 400;
                DemoObject obj = ObjectSpace.FindObject<DemoObject>(
                    CriteriaOperator.Parse(
                    string.Format("Width = '{0}' and Height = '{1}'", width, height)));
                if (obj == null) {
                    obj = ObjectSpace.CreateObject<DemoObject>();
                    obj.Width = width;
                    obj.Height = height;
                }
            }
            ObjectSpace.CommitChanges();
        }
    }
}

File: VB/PopupSizeExample.Module/DatabaseUpdate/Updater.vb
Imports System

Imports DevExpress.ExpressApp
Imports DevExpress.ExpressApp.Updating
Imports DevExpress.Xpo
Imports DevExpress.Data.Filtering
Imports DevExpress.Persistent.BaseImpl
Imports DevExpress.ExpressApp.Security
Imports PopupSizeExample.Module.BusinessObjects

Namespace PopupSizeExample.Module.DatabaseUpdate
    Public Class Updater
        Inherits ModuleUpdater

        Public Sub New(ByVal objectSpace As IObjectSpace, ByVal currentDBVersion As Version)
            MyBase.New(objectSpace, currentDBVersion)
        End Sub
        Public Overrides Sub UpdateDatabaseAfterUpdateSchema()
            MyBase.UpdateDatabaseAfterUpdateSchema()
            For i As Byte = 0 To 10
                Dim width = i * 40 + 600
                Dim height = i * 20 + 400
                Dim obj As DemoObject = ObjectSpace.FindObject(Of DemoObject)(CriteriaOperator.Parse(String.Format("Width = '{0}' and Height = '{1}'", width, height)))
                If obj Is Nothing Then
                    obj = ObjectSpace.CreateObject(Of DemoObject)()
                    obj.Width = width
                    obj.Height = height
                End If
            Next i
            ObjectSpace.CommitChanges()
        End Sub
    End Class
End Namespace
